docs/content/preview/yugabyte-platform/prepare/server-nodes-software/software-on-prem-manual.md

```diff
@@ -142,6 +142,15 @@ Physical nodes (or cloud instances) are installed with a standard AlmaLinux 8 se
 
     Ensure that the `yugabyte` user has permissions to SSH into the YugabyteDB nodes (as defined in `/etc/ssh/sshd_config`).
 
+    If you are using a CIS hardened image and want SSH access to database nodes, you need to manually add the `yugabyte` user to `sshd_config`.
```

Comment:
Is this applicable to any CIS hardened image we support - Amazon Linux 2
(ARM, CIS-hardened) and RHEL 8 and 9 ?